packages/design-system-docs/src/pages/components/TextField/Mask.example.jsx

```diff
@@ -76,7 +75,6 @@ const Example = () => {
         label="Social security number (SSN)"
         mask="ssn"
         inputMode="numeric"
```

Comment:
Why are these patterns being removed for ssn and zip? They do look like they should be updated to match the values in `Mask.jsx` though

Reply:
I found in my testing that the pattern attribute gets added automatically and doesn't need to be part of the example. Setting the `TextField` mask to="ssn' automatically sets the pattern to `[0-9-*]*` Mask.jsx

Reply:
Ah that makes sense thank you.